Dev/VideoChatClient/ClientStartup/ClientStartup.cpp

```diff
@@ -18,21 +18,21 @@ uint32_t ClientStartup::send_client_hello(Socket& socket, const std::string& use
 {
 	PacketCreator packet_creator{UNKNOWN_MEETING_ID, username, UNKNOWN_USER_ID};
 
-	packet_creator.create(PacketUtils::PacketTypes::ClientHelloPacketType, {});
+	packet_creator.create(PacketUtils::PacketTypes::ClientHello, {});
 	socket << packet_creator.get_raw_packet();
 
 	PacketUtils::PacketHeaders headers{};
 	socket >> headers;
-	PacketUtils::validate_packet(headers, UNKNOWN_MEETING_ID);
+	PacketUtils::validate_packet(headers, UNKNOWN_MEETING_ID, PacketUtils::PacketTypes::ServerHello);
```

Comment:
you are passing UNKOWN_MEETING_ID which is -1 to a uint32_t, what will happen?

Reply:
UNKNOWN MEETING is a unit32_t